interview-questions.md

```diff
@@ -8,48 +8,48 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX is the coding format for React that allows the code to be written in html and javascript. a syntax difference is the use of className to modify an element in JSX unlike html whihc uses class to modify an element. 
 
-Researched answer:
+Researched answer: JSX is a syntax extension of Javascript which behaves very similarly to html, It also uses a syntax called classNAme instead of class like html to modift elements because class is a protected javascript syntax word.
```

Comment:
This would be a great place to expand on what you already know and provide more context.